Makefile

```diff
@@ -4,7 +4,8 @@ LDFLAGS = -pthread -lncurses
 BINS = tetris
 all: $(BINS)
 
-OBJS = main.o ui.o game.o
+SRCS = $(wildcard *.c)
```

Comment:
Since the project was designed for lecture assignment, there might be several source files in the same directory. It would make confusion if students intend to switch over different implementations.

Reply:
ok